### ISSUE TITLE
Channel Moderation

### ISSUE THREAD

**Issue by AidanTheBandit** [syuilo/Misskey#9126]
## Summary

Hello! I've been working on a beta for my Misskey instance that allows channel owners to have moderation features in their channel like deleting messages, setting channel join ability (private and public), setting channel moderators,ban, mutes, and timeouts. I think this would also be useful to other misskey users though since it would help make channels nicer to use

**Comment by mateo787** [comment]
yes i desperately need this feature,Because the current version does not have channel deletion permissions, some users create channels at will
How do I use this beta version?

**Comment by AidanTheBandit** [comment]
It's not out yet. I should have it out on my fork of Misskey soon. I don't know if it will support federation (I removed fedi from my Misskey instance so I can add new features). It should be out soon within the coming months though
